gapic/api/artman_datastore.yaml

```diff
@@ -0,0 +1,25 @@
+common:
+  api_name: google-datastore-v1
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    - ${REPOROOT}/googleapis/google/datastore/v1
+  service_yaml:
+    - ${REPOROOT}/googleapis/google/datastore/datastore.yaml
+  output_dir: ${REPOROOT}/artman/output
+  gapic_api_yaml:
+    - ${REPOROOT}/googleapis/google/datastore/v1/datastore_gapic.yaml
+java:
+  final_repo_dir: ${REPOROOT}/gcloud-java/gcloud-java-datastore
```

Comment:
This will need to change to ${REPOROOT}/google-cloud-java/google-cloud-datastore (I'm changing the existing ones in my own googleapis PR)